rajawali/src/main/java/org/rajawali3d/util/egl/RajawaliEGLConfigChooser.kt

```diff
@@ -0,0 +1,75 @@
+package org.rajawali3d.util.egl
+
+import android.annotation.TargetApi
+import android.opengl.EGLExt
+import android.opengl.GLSurfaceView
+import android.os.Build
+import org.rajawali3d.view.ISurface
+
+import javax.microedition.khronos.egl.EGL10
+import javax.microedition.khronos.egl.EGLConfig
+import javax.microedition.khronos.egl.EGLDisplay
+
+/**
+ * @author Jared Woolston (jwoolston@tenkiv.com)
+ */
+class RajawaliEGLConfigChooser(glMajorVersion: Int, val antiAliasingConfig: ISurface.ANTI_ALIASING_CONFIG,
```

Comment:
Bring all the parameters to their own line. Anytime you have more than 3, it's far easier to read broken up. AS as an intention to do it for you just fyi.